client/test-worker/turbo-resolver/resolver.spec.js

```diff
@@ -24,3 +24,18 @@ test('Resolver resolves vue2 deps', async t => {
   t.deepEqual(Object.keys(result.appDependencies), ['vue']);
   t.equal(Object.keys(result.resDependencies).length, 0);
 });
+
+test('Resolver resolves invalid deps', async t => {
+  t.timeoutAfter(5000);
+
+  const r = new Resolver(new NpmHttpRegistry());
+
+  try {
+    await r.resolve({
+      'aninvalidmodulename': '^2.0.0'
+    });
+    throw new Error('must failed');
```

Comment:
This error will be caught by next line.
Use `t.fail('...')` instead.

Reply:
Yes I know, I check the error. Thanks for the t.fail tips.
I tried to simply use `t.rejects` but without success.